user_interaction/templates/user_interaction/account_pages/account_edit.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'core/user_accounts/account_base.html' %}
+{% load generic_field %}
+
+{% block title %}
+  Squire - Edit Account
+{% endblock title %}
+
+{% block content %}
+    <h1>Edit Account Information</h1>
+    <form method="post">
+        {% csrf_token %}
+        {% if form.non_field_errors %}
+          <ul class="errorlist alert alert-danger">
+            {% for error in form.non_field_errors %}
+              <li>{{ error }}</li>
+            {% endfor %}
+          </ul>
+        {% endif %}
+
+            {% generic_field form.username -1 %}
+
+            {% generic_field form.first_name -1 %}
+
+            {% generic_field form.email -1 %}>
```

Comment:
There's an additional `>` at the end
```suggestion
            {% generic_field form.email -1 %}
```